src/preprocess/sql_generation/sql_common.py

```diff
@@ -0,0 +1,38 @@
+import re
+
+
+def write(f, text):
+    # print(text)
```

Comment:
Leftover debug line?

Reply:
Originally it wasn’t commented out, but Jordi asked me to comment it out back some years ago. It can actually be removed, although it’s a good spot to check the logs if debugging is needed.

Reply:
I have fixed this one